Pod/Classes/PIDatePicker.swift

```diff
@@ -67,11 +67,6 @@ public class PIDatePicker: UIView, UIPickerViewDataSource, UIPickerViewDelegate
 
     // MARK: -
     // MARK: LifeCycle
-    
-    override public init() {
-        super.init()
-        self.commonInit()
-    }
```

Comment:
check the `Initializer Delegation for Class Types` section of https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Initialization.html